### PR TITLE
fix kosmos-scheduler miss lease permission

### PR DESCRIPTION
**Pull request by qiuming520**
#### What type of PR is this?

- **fix:** fix bug
#### What does this PR do?

add parameter for kosmos-scheduler
--leader-elect-resource-name=kosmos-scheduler
--leader-elect-resource-namespace=kosmos-system
kosmos-scheduler add lease permission

#### Which issue(s) does this PR fix?

Fixes #https://github.com/kosmos-io/kosmos/issues/117
#### Screenshots (if applicable):

```release-note

```

#### Additional Comments:

```release-note
fix bug kosmos-scheduler compete for lease -- kube-system/kube-scheduler
```

### CODE REVIEW

File: deploy/scheduler/rbac.yaml [hunk 1]
```diff
@@ -49,6 +49,7 @@ rules:
       - leases
     resourceNames:
       - kube-scheduler
+      - kosmos-scheduler
```
Comment (duanmengkk) [inline deploy/scheduler/rbac.yaml:51]: should remove the kube-scheduler ？

Comment (qiuming520) [reply deploy/scheduler/rbac.yaml:51]: yes, kube-scheduler is no longer needed

**Pull request merged**